frontend/src/features/books/BookDetails.jsx

```diff
@@ -1,27 +1,22 @@
 import React, { useEffect, useState } from 'react'
-import { useParams, Link } from 'react-router-dom'
+import { useParams, Link, useNavigate } from 'react-router-dom'
 import { API_URL } from '../../constants'
 
 function BookDetails() {
```

Comment:
@nshelokar-vf, is there any specific reason for using functional components and not preferring arrow functions for components?

Reply:
No sir, there is not any specific reason for that, I am little familiar with traditional functions. But I will make sure to use arrow functions from next time.